### PR TITLE
Fix error output on storage list

### PR DESCRIPTION
**Pull request by simonswine**

### CODE REVIEW

File: server/handlers.go [hunk 1]
```diff
@@ -224,8 +224,8 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 	}
 
 	connectors, e := s.storage.ListConnectors()
-	if e != nil {
-		s.logger.Errorf("Failed to get list of connectors: %v", err)
+	if err != nil {
```
Comment (srenatus) [inline server/handlers.go]: Should this not stay `if e != nil {`? Looks like you've fixed the bug twice 😉

Comment (simonswine) [reply server/handlers.go]: Well that's embarrassing. I have fixed it in the latest push

**Comment by srenatus** [comment]
Looks like we have some issue with the Travis build that's unrelated to this change...

**Comment by srenatus** [comment]
Huh! And I thought I had seen that before when merging #1443. Sorry, @simonswine, that your contribution fell off the map here. On the positive side, the correct error is printed now 🎉 😉

**Pull request closed without merging**